easyconfigs/p/picard/picard-2.27.5-Java-11.0.18.eb

```diff
@@ -0,0 +1,60 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+# 2.18.11:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+# 2.21.1
+# Pavel Grochal (INUITS)
+# 2.25.1
+# J. Sassmannshausen (GSTT)
+# 2.25.5
+# Erica Bianco (HPCNow!)
+# 2.26.10
+# Christoph Siegert (Leipzig University)
+# 3.0.0
+# Graham Derryberry (UTK)
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.27.5'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://broadinstitute.github.io/picard/'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = [{
+    'filename': '%(name)s-%(version)s.jar',
+    'download_filename': '%(name)s.jar',
+}]
+checksums = ['131ca3e0626a3ef12e8399795e7a49f829bb4e85d71b489bf4f231d21a6c3199']
+
+postinstallcmds = ["mv %(installdir)s/%(name)s-%(version)s.jar %(installdir)s/%(name)s.jar"]
+
+dependencies = [('Java', '11.0.18')]
```

Comment:
Should this use the Java wrapper here i.e ('Java', '11'), as SB suggested I use in this older PR - https://github.com/bear-rsg/bear-eb/pull/233#pullrequestreview-1641807404

Reply:
No as the wrapper leads to Java 11.0.2 which is not compaoble with the Java we have with R, @branfosj are you happy for us to have this?